### ISSUE TITLE
inconsistent position

### ISSUE THREAD

**Issue by 29vd** [Clinical-Genomics/scout#480]
c.DNA and protein position is not consistent with genome position in this specific transcript in SOX5
NM_006940 - g.23908582 is according to Scout c.519C>T, p.Gly173=, men är c.558C>T p.Gly186=

**Comment by moonso** [comment]
Ok this is an issue for VEP @henrikstranneheim . We do not calculate these positions, we only collect them from VEP

**Comment by henrikstranneheim** [comment]
I will take a 👀 tomorrow 

Skickat från min iPhone

**Comment by henrikstranneheim** [comment]
I do not know where this occurs and therefore I cannot investigate.

It would be great if you always could include a link to the page where the issue occurs.

**Comment by dnil** [comment]
And I still think it is not a tremendous idea to populate the git threads with actual genetic variants, rather try to keep them in tickets instead and discuss general matters in issues? Anyway, the cat is out of the box on this one so here goes: 
https://clinical-db.scilifelab.se:8083/cust002/F0002632/164642a9b2a2e3743dacc21c44d8eda1

I couldn't help taking a :eyes: too. Seems to go back to the transcript resource. Mapping between ENSMBL and RefSeq isn't going too swell here. The ENSEMBL is quite confused on the mapping.  ENST00000309359 - encoding a 750aa peptide -  is noted as mapping to NM_006940 and NM_152989. However NCBI and RefSeq is quite convinced NM_006940 is a transcript coding for a 763 aa peptide. Still, as long as a refSeq transcript set is used for annotation, this should be fine?

**Comment by henrikstranneheim** [comment]
Yeah, that might be prudent, but enough information to be able to loacate the issue is obviously necessary. Customer, case and gene should suffice for most issues.

**Comment by moonso** [comment]
Great! Please close the issue if you can agree on this @29vd and @dnil

**Comment by dnil** [comment]
We probably need to discuss transcript sets again. While I love to have the more complete ENSMBL set, that should not be at the expense of correct use of refSeq. Personally I'd then vote for switching to refseq completely if we can't figure out a better way for them to coexist.

**Comment by henrikstranneheim** [comment]
Take a look at REFSEQ_MATCH here:
http://www.ensembl.org/info/docs/tools/vep/vep_formats.html

Unfortunately available for GRCh37, but it seems to map discrepancies between the transcript builds.

**Comment by dnil** [comment]
We should probably get to implementing this. Had another one today which caused a lot of discussion in the lab. 
https://clinical-db.scilifelab.se:8083/cust002/F0009777/2d213a6ad4e157c89573154fa4f7800c
Note how the designated canonical refseq transcript from HGNC is not available in the ENSEMBL GRCh37 build (and so not in Scout), but then the two RefSeq transcripts are also incorrectly assigned as a single transcript in ENSEMBL GRCh38, although they differ by a few amino acids in length. If VEP already knows about this, it would be pretty good to display it!

**Comment by moonso** [comment]
Could anyone @dnil @henrikstranneheim suggest a solution to this problem?
If it should be done in MIP, please close this and move there. Otherwise we need to understand exactly how to solve it

**Comment by dnil** [comment]
This is essentially MIP / best practice policy issue, not a Scout - Scout one. I have a feeling @henrikstranneheim might have enabled REFSEQ_MATCH in VEP already? Lets move it to MIP.
